sonic-net/sonic-buildimage
# [Accton/AS4630] update reg use for new CPLD FW

Models : AS4630-54PE, AS4630-54TE

1. Update CPLD reg use: fan_dir, LEDs(POE, DIAG, PSU1/2, FAN) and
    PSU power_good
2. ~~Improve the PSU model/serial_num reading to avoid illegal char.~~ Done by [PR#5389](https://github.com/Azure/sonic-buildimage/pull/5389)
3. Correct typos in cpld.c

Signed-off-by: Sean Wu <sean_wu@edge-core.com>

#### Why I did it
Change registers use according to the newly released CPLD FW spec.

#### How I did it
Change registers use according to the newly released CPLD FW spec.

#### How to verify it
1. LEDs: cat brightness of those modified LEDs, and check them visually at the same time.
2. PSU models/serial num: `show platform psustatus`, check if there are no illegal characters within.
```
root@tor:/sys/bus/i2c/devices/3-0060# show platform psustatus 
PSU    Model       Serial              HW Rev      Voltage (V)    Current (A)    Power (W)  Status    LED
-----  ----------  ------------------  --------  -------------  -------------  -----------  --------  -----
PSU 1  YPEB1200AM  SB020X181951000044  N/A               54.50           0.00         0.00  OK        green
PSU 2  YPEB1200AM  SB020X181951000017  N/A               54.44           0.53        29.00  OK        green
```

#### Which release branch to backport (provide reason below if selected)


#### Description for the changelog
[Accton/AS4630] update reg use for new CPLD FW

#### A picture of a cute animal (not mandatory but encouraged)
